cmd/influxd/inspect/dump_tsm/dump_tsm_test.go

```diff
@@ -171,7 +171,7 @@ func makeArgs(path string, meas string) (args map[string][]string) {
 	args[argsKeys[1]] = []string{"--file", path, "--blocks"}
 	args[argsKeys[2]] = []string{"--file", path, "--all"}
 	if meas != "" {
-		args[argsKeys[3]] = []string{"--file", path, "--filter-key", meas}
+		args[argsKeys[3]] = []string{"--file", path, "--filter-key", meas, "--all"}
```

Comment:
Is this change also needed, or was the `Flush()` the only thing?

Reply:
It may not be needed, but it shouldn't cause any harm to keep. I can attempt to remove it if you would like

Reply:
Nah it's ok by me to leave in, was just curious